package.json

```
@@ -9,7 +9,15 @@
   "module": "./dist/fuse.esm.js",
   "unpkg": "./dist/fuse.js",
   "jsdelivr": "./dist/fuse.js",
-  "typings": "./dist/fuse.d.ts",
+  "types": "./dist/fuse.d.ts",
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./dist/fuse.d.ts",
+      "import": "./dist/fuse.esm.js",
+      "require":  "./dist/fuse.common.js"
```

```suggestion
      "import": {
        "types": "./dist/fuse.d.ts",
        "default": "./dist/fuse.esm.js"
      },
      "require":  {
        "types": "./dist/fuse.d.ts",
        "default": "./dist/fuse.common.js"
      }
```
see https://github.com/microsoft/TypeScript/issues/50466

---

updated! Thanks for reviewing!